fury/primitive.py

```diff
@@ -851,3 +852,223 @@ def prim_cylinder(radius=0.5, height=1, sectors=36, capped=True):
         triangles = (np.array(triangles).reshape(2 * sectors, 3))
 
     return vertices, triangles
+
+
+def init_parametric(u_lower_bound, u_upper_bound, v_lower_bound,
+                    v_upper_bound, npoints):
+    u = np.linspace(u_lower_bound, u_upper_bound, npoints)
+    v = np.linspace(v_lower_bound, v_upper_bound, npoints)
+    u, v = np.meshgrid(u, v)
+    u = u.reshape(-1)
+    v = v.reshape(-1)
+    return u, v, np.sin, np.cos, np.mean
+
+
+def prim_para_mobius_strip(npoints=100):
+    """Return vertices and triangle for a Möbius strip
+
+    Returns
+    -------
+    vertices: ndarray
+        vertices coords that compose our Möbius strip
+    triangles: ndarray
+        triangles that compose our Möbius strip
+    """
+
+    u, v, sin, cos, mean = init_parametric(0, 2*np.pi, -1, 1, npoints)
+    points2D = np.vstack([u, v]).T
+    tri = Delaunay(points2D)
+    triangles = tri.simplices
```

Comment:
duplicated in all function

Reply:
Reduced the duplication by making build_parametric function